meson.build

```diff
@@ -44,9 +44,28 @@ elif bpf_clang_maj < 17
           .format(bpf_clang.full_path(), bpf_clang_ver))
 endif
 
-if get_option('libbpf_a') != ''
+libbpf_a = '@0@/libbpf/src/libbpf.a'.format(meson.current_source_dir())
+
+if get_option('libbpf_a') == 'disabled'
+  libbpf_a = ''
+elif get_option('libbpf_a') != ''
+  libbpf_a = get_option('libbpf_a')
+endif
+
+if libbpf_a != ''
+  if not fs.exists(libbpf_a)
```

Comment:
So, we end up running this once. Wouldn't it be better to have a more permanent way of detecting the default case (it can just be a bool from the above if/else block too) and run make each time?

Reply:
Good call. If people pull and we've update the submodule we'd want to run it again.